app/indexers/administrative_tag_indexer.rb

```diff
@@ -18,17 +18,20 @@ def initialize(id:, administrative_tags:, **)
   def to_solr
     Rails.logger.debug { "In #{self.class}" }
 
-    solr_doc = { 'tag_ssim' => [], 'exploded_tag_ssim' => [] }
+    solr_doc = { 'tag_ssim' => [], 'exploded_tag_ssim' => [], 'exploded_nonproject_tag_ssim' => [] }
     administrative_tags.each do |tag|
+      tag_prefix, rest = tag.split(TAG_PART_DELIMITER, 2)
+      prefix = tag_prefix.downcase.strip.gsub(/\s/, '_')
```

Comment:
`tag_prefix` and `prefix` could probably have less ambiguous names (`raw_tag_prefix` and `tag_prefix`?  `tag_prefix` and `normalized_tag_prefix`?).  but for now, i just moved the declarations up a few lines, since i needed what is currently `prefix` before the `next if SPECIAL_TAG_TYPES_TO_INDEX.exclude? ...` after which it was being set.